### PR TITLE
Add page nav footer to every page.

### LINKED ISSUE #31: Add other audience sections to footer of each page
The homepage has this footer:

screen shot 2017-04-06 at 1 55 13 pm

After clicking to the other pages, I kind of expected to see the same footer again. But links to other pages are only in the nav.

Does it make sense to add the footer to the other two audience pages, too?

cc @MikeMcQuaid

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This makes it more obvious how to navigate between pages when you're done reading one. This was already present on the index.

Fixes #31.

### CODE REVIEW

File: app/views/home/contributors.html.erb [hunk 1]
```diff
@@ -216,5 +216,23 @@
...
       </div>
     </div>
+
+    <div class="pt-6">
+      <div class="clearfix d-md-flex gut-md my-6">
+        <div class="col-md-6">
+          <h2 class="alt-h3 my-2">
+            <a href="/" class="text-blue">Businesses <%= octicon 'chevron-right', height: '26', class: 'text-inherit' %></a>
+          </h2>
+          <p class="text-gray alt-body-font">Your business needs to use and contribute to open source software to run effectively.</p>
```
Comment (MikeMcQuaid) [inline app/views/home/contributors.html.erb]: @nayafia Thoughts on this wording?

Comment (nayafia) [inline app/views/home/contributors.html.erb]: Maybe

> "Your business runs on open source. Learn how to get involved."

? It sounds less like you're scolding them :wink:

File: app/views/home/contributors.html.erb [hunk 2]
```diff
@@ -216,5 +216,23 @@
...
+        </div>
+
+        <div class="col-md-6">
+          <h2 class="alt-h3 my-2">
+            <a href="/maintainers" class="text-red">Maintainers <%= octicon 'chevron-right', height: '26', class: 'text-inherit' %></a>
+          </h2>
+          <p class="text-gray alt-body-font">Get your project ready for the flow of accepting and encouraging regular new contributors.</p>
```
Comment (nayafia) [inline app/views/home/contributors.html.erb]: > Give your project some love to help others contribute where you need it most.

or something a little more happy/aspirational :blush:

File: app/views/home/maintainers.html.erb [hunk 3]
```diff
@@ -141,4 +141,25 @@
...
+          <h2 class="alt-h3 my-2">
+            <a href="/" class="text-red">Businesses <%= octicon 'chevron-right', height: '26', class: 'text-inherit' %></a>
+          </h2>
+          <p class="text-gray alt-body-font">Your business needs to use and contribute to open source software to run effectively.</p>
+        </div>
+        <div class="col-md-6">
+          <h2 class="alt-h3 my-2">
+            <a href="/contributors" class="text-blue">Contributors <%= octicon 'chevron-right', height: '26', class: 'text-inherit' %></a>
+          </h2>
+          <p class="text-gray alt-body-font">Learn from people all over the world how the software you already use is built and build your reputation in the process.</p>
```
Comment (nayafia) [inline app/views/home/maintainers.html.erb]: A bit more concise:

> Build your reputation while collaborating on software with people around the world.

**Comment by MikeMcQuaid** [comment]
Thanks @nayafia. I do love a good scolding, though 😉

**Pull request merged**